crates/primitives/src/hex_bytes.rs

```diff
@@ -318,4 +357,17 @@ mod tests {
         let wrong_b = bytes::Bytes::from("0123absd");
         assert_ne!(wrong_b, b);
     }
+
+    #[test]
+    fn arbitrary() {
+        proptest::proptest!(|(bytes: Bytes)| {
```

Comment:
I think there's a marginally more compact syntax for all these where you can do `fn arbitrary(bytes: Bytes)` and wrap the test with a proptest like https://docs.rs/proptest/latest/proptest/macro.proptest.html#example, but nbd.